numba/tests/test_unicode.py

```diff
@@ -337,6 +349,41 @@ def test_find(self, flags=no_pyobj_flags):
                                  cfunc(a, substr),
                                  "'%s'.find('%s')?" % (a, substr))
 
+    def test_rfind(self):
```

Comment:
It's probably worth checking the CPython `rfind` tests for some further inspiration https://github.com/python/cpython/blob/1d4b6ba19466aba0eb91c4ba01ba509acf18c723/Lib/test/test_unicode.py#L232-L258

Reply:
Sure. I'll improve the testing.